recipe/0046-rendering/index.md

```diff
@@ -1,43 +1,42 @@
+summary: "Linking to non-IIIF representations of the object, such as a PDF."
 ---
 
-
 ## Use Case
 
-Why is this pattern is important?
+You have an archival object in IIIF format for which you would like to offer researchers the opportunity to download a PDF version. You know from previous experience that researchers using your collection like to have images and texts available for offline reading, or you know you have patrons with bandwidth concerns who need to minimize their time online, or other reasons such as wanting to provide transcriptions and translations of objects' text. Through use of the `rendering` property, you are able to alert conforming clients to the presence of this other format (or yet other formats, such as TEI) so they in turn can provide appropriate UX workflows to users.
 
 ## Implementation notes
 
-How does one implement the pattern?
+This property is used for pointing a researcher to a non-IIIF representation of the resource to which it is attached. This is the biggest distinction between this property and `accompanyingCanvas`, which might seem functionally similar. In addition, this property addresses additional representations of the same resource while `accompanyingCanvas` is used for content complementary to the main resource and rendered simultaneously.
+
+Any resource may have the `rendering` property. Each instance of it may have multiple items contained inside it, provided each has the `id`, `type`, and `label` properties at a minimum. Conforming clients should render this property in some way on a Collection, Manifest, or Canvas, but may on other resources. Keep in mind that the client's display of the property has no defined form.
 
 ## Restrictions
 
-When is this pattern is usable / not usable? Is it deprecated? If it uses multiple specifications, which versions are needed, etc.? (Not present if not needed.)
+This property is not for use for presenting multiple IIIF versions of the same view, such as an infrared and natural light view of a painting. For that use case, you should use [Choice (simplest)][0033] or [Choice — multispectral flavored example, with image services][0034].
 
 ## Example
 
-Describe in prose and provide examples, e.g.: 
+In this example, the PDF to be made available is for the program as a whole, and as such the `rendering` property is on the Manifest. If PDFs were available for each view or page separately, each Canvas would be the logical place for the corresponding PDF.
 
-``` json-doc
-{
-  "@context": [
-    "http://www.w3.org/ns/anno.jsonld",
-    "http://iiif.io/api/presentation/{{ page.major }}/context.json"
-  ],
-  "id": "https://example.org/iiif/book1/manifest",
-  "type": "Manifest" 
-}
-```
+To see the property in actiono in Mirador, toggle the sidebar by activating the three-line ("hamburger") menu in the upper left-hand corner of the content window. You should then, in the "Related" area, see the link under the "Alternate formats" heading.
```

Comment:
Typo: "actiono" -> "action"

Reply:
Fixed in https://github.com/IIIF/cookbook-recipes/pull/214/commits/fb5709e5ad054c31d2fe1c36656f06e251134317